packages/typespec-client-generator-core/test/types.test.ts

```diff
@@ -151,15 +151,9 @@ describe("typespec-client-generator-core: types", () => {
         @access(Access.public)
         model Test {
           urlScalar: url;
-          uuidScalar: uuid;
-          eTagScalar: eTag;
 
           @format("url")
           urlProperty: string;
-          @format("uuid")
-          uuidProperty: string;
-          @format("eTag")
-          eTagProperty: string;
```

Comment:
`uuid` and `eTag` are azure types, we should not support their format by default.